lib/github.js

```diff
@@ -71,6 +71,12 @@ Organization.prototype.getRepositories = function(callback) {
             cli.debug('next page ' + this.nextPageUrl);
             cli.debug('last page ' + this.getLastPage());
 
+            if ( body.message ) {
```

Comment:
I've checked further and saw, that GitHub indeed sends HTTP error codes together with REST responses. But cloneorg somehow sees them as regular replies (no error), so that error check currently works, but the problem should be solved using another way ;)